README.md

```diff
@@ -662,6 +662,7 @@ extensions = {'quickfix'}
 - quickfix
 - symbols-outline
 - toggleterm
+- aerial
```

Comment:
the list is alphabetically sorted so aerial goes on top .

Reply:
Already adjusted it. I only modified README because the doc can be generated automatically. Thanks.

Reply:
> I only modified README because the doc can be generated automatically

You should remove changes from doc/lualine.txt . I think CI is hitting merge conflict :D

Reply:
Removed already. Thank you.